.mvn/local-settings.xml

```diff
@@ -1,9 +1,8 @@
-<!--
-    Derived from: https://stackoverflow.com/a/67002852
--->
 <settings xmlns="http://maven.apache.org/SETTINGS/1.2.0"
           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
           xsi:schemaLocation="http://maven.apache.org/SETTINGS/1.2.0 http://maven.apache.org/xsd/settings-1.2.0.xsd">
+<!--
+    Derived from: https://stackoverflow.com/a/67002852
```

Comment:
This section can be removed

Reply:
I thought it might be a good idea to keep it commented out in case we need it for something else again but I guess that's not very likely :)